### DISCUSSION TITLE
Gossamer + Gosemble for developing parachains in Go?

### DISCUSSION THREAD

**Discussion by radkomih** [ChainSafe/gossamer#3531] [Research]
Hi there,

We have a working implementation of a Go-based framework, [Gosemble](https://github.com/LimeChain/gosemble/tree/develop), capable of producing Polkadot-compatible Runtimes. Currently, Gosemble supports only a few pallets, but we're working on adding more. Our aim is to create a full Go-based stack for writing parachains and we're interested in using Gossamer as a host for Gosemble runtimes. At the moment, we're using Gossamer solely for running integration tests, while employing a Substrate-based host for running a network.

Would you be interested in such an integration? We'd love to discuss this further, learn more about your plans, roadmap, etc., and also share our current status and future plans.

**Comment by danforbes** [comment]
Thank you so much for creating this Discussion 🙏🏻 I am going to follow up offline to provide you all with a scheduling link so that we can meet via video conference and discuss this awesome idea in greater detail 🚀
